server/sonar-web/src/main/js/app/components/nav/global/GlobalNav.tsx

```diff
@@ -158,4 +158,4 @@ const mapStateToProps = state => {
 
 const mapDispatchToProps = { skipOnboarding };
 
-export default connect(mapStateToProps, mapDispatchToProps)(GlobalNav);
+export default connect<StateProps, DispatchProps>(mapStateToProps, mapDispatchToProps)(GlobalNav);
```

Comment:
I see only one benefit of this solution: to be able to use `ownProps.location` inside `mapStateToProps`. But because I don't need it (and moreover I should not depend on location), I'd not use it.

Reply:
Until now I only thought about `OwnProps` as the props that needs to be passed down to the container component when using it.
I never really thought about what I was using or not in the `mapStateToProps` when defining it.